src/services/impl/GameContext.cpp

```diff
@@ -0,0 +1,29 @@
+#include "GameContext.hh"
+
+#include "RA_Log.h"
+#include "RA_StringUtils.h"
+
+#include "services\impl\FileTextReader.hh"
+#include "services\impl\FileTextWriter.hh"
+
+namespace ra {
+namespace services {
+namespace impl {
+
+void GameContext::LoadGame(unsigned int nGameId, const std::wstring& sGameTitle)
```

Comment:
It's up to you but this looks like a constructor.

Reply:
For now. In the future, it'll make the server hit to acquire the data and populate the fields from the response.

Reply:
Also, it has to be able to change when the player switches from one game to another.